scripts/ci.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# Invoke xcodebuild
+#
+# \param $1 type of project (Project or Workspace)
+# \param $2 path to  to run against
+# \param $3 scheme to build
+# \param $4 configuration
+# \param $5 sdk
+# \param $6+ build commands
+function invoke_xcodebuild()
+{
+    /usr/bin/xcodebuild \
+        -"$1" "$2" \
+        -scheme "$3" \
+        -configuration "$4" \
+        -sdk "$5" \
+        "${@:6}" \
+        CODE_SIGNING_ALLOWED=NO
+    
+    return $?
+}
+
+# Run a macOS build and test with the specified configuration
+# 
+# \param $1 configuration
+function macos_build_test()
+{
+    invoke_xcodebuild project "macos/xcode/OfficeUIFabric.xcodeproj" OfficeUIFabricTestApp "$1" macosx build test
+}
+
+# Run an iOS simulator xcodebuild invocation with the specified scheme, configuration, and build commands
+#
+# \param $1 scheme
+# \param $2 configuration
+# \param $3+ build commands
+function ios_simulator()
+{
+    invoke_xcodebuild workspace "ios/OfficeUIFabric.xcworkspace" "$1" "$2" iphonesimulator "${@:3}" -destination "platform=iOS Simulator,name=iPhone 8"
+}
+
+# Run an iOS device xcodebuild invocation with the specified scheme and configuration
+#
+# \param $1 scheme
+# \param $2 configuration
+function ios_build_device()
```

Comment:
very minor, but above in macos_build_test we establish the pattern of {platform}_{actions}. This one feels like {platform}_{actions}_{platformPart2}. Not sure what the proper answer is, but should be consistent